src/dos/dos.cpp

```diff
@@ -184,6 +218,32 @@ static void DOS_AddDays(Bitu days)
 	}
 }
 
+std::vector<std::string> FILE_EGA_CPX = {
+	"EGA.CPX", "EGA2.CPX", "EGA3.CPX", "EGA4.CPX", "EGA5.CPX",
+	"EGA6.CPX", "EGA7.CPX", "EGA8.CPX", "EGA9.CPX", "EGA10.CPX",
+	"EGA11.CPX", "EGA12.CPX", "EGA13.CPX", "EGA14.CPX",
+	"EGA15.CPX", "EGA16.CPX", "EGA17.CPX", "EGA18.CPX",
+};
+
+std::vector<std::string> FILE_KEYBOARD_SYS = {
+	"KEYBOARD.SYS", "KEYBRD2.SYS", "KEYBRD3.SYS", "KEYBRD4.SYS",
+};
```

Comment:
Moving these to `constexpr` std::array's (so we still get a `.size()` to use in the for loops), appears to be OK, and also avoids extra copying into `std::string` objects, and we can avoid the `.c_str()` in a bunch of places:

![2022-03-29_22-53](https://user-images.githubusercontent.com/1557255/160761110-315a3e03-8e4f-48af-ba28-8f6ac409a673.png)